EncoreTickets.SDK/Basket/Extensions/ReservationExtension.cs

```diff
@@ -1,10 +1,25 @@
-﻿using EncoreTickets.SDK.Basket.Models;
+﻿using System;
+using EncoreTickets.SDK.Basket.Models;
+using EncoreTickets.SDK.Utilities.CommonModels.Constants;
 using EncoreTickets.SDK.Utilities.CommonModels.Extensions;
 
 namespace EncoreTickets.SDK.Basket.Extensions
 {
     public static class ReservationExtension
     {
+        /// <summary>
+        /// Indicates whether the reservation is used to flag flexi tickets.
+        /// </summary>
+        /// <param name="reservation"></param>
+        /// <returns></returns>
+        public static bool IsForFlexiTickets(this Reservation reservation)
```

Comment:
From the usage it seems that 'IsFlexiTicket' or 'IsFlexi' is a better name for this